### ISSUE TITLE
WCAG issues with deactivated and read-only form components

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#29]
## Description
As this is a useful and common state for components to be in, it has some accessibility and usability issues.

## Considerations
1. They are harder to read and they do not get focus when tabbing trough the form.
This is not an issue in cases where you do not want the user to focus on the component, but if it is a read-only field that has data that is going to be submitted as a part of the form. In these cases is a problem that some users might have a problem perceive the data. 

2. Deactivated fields do not inherently tell the user why they are deactivated. hiding the input and if applicable give the user a text hint too why they cant choose something might be a better way to solve this. 

## Tasks
- [ ] If we are not coding these as inputs anymore, they will not have too look like inputs and they will not have labels and description. Does this change the design in any way?
- [ ] we still have the greyed out fields with the `readonly` attribute, they are breaking WCAG with to low contrast. How do we handle them? 
- [ ] make sure the design and changes handle the calculater in "oppstartsveilederen"
- [ ] document (Guidelines and documentation) how to set up prefill/readonly data for the user.

## Technical discussion
We need to discuss with the developers what the component should be. There are some accessibility concerns about read-only fields, and i'm unsure on what is the best way to go about this. 
- [x] What should this component be? An input field with a read-only tag or should we make this just plain text?
    - I think we landed on presenting it as text?

## Acceptance criteria
The following criteria are relevant if we choose to solve this with read-only form components
- [ ] The field should not get focus when tabbing
- [ ] Screen reader support
- [ ] Should be clear from the visual design that the user cant edit the field
- [ ] Should be able to handle prefilled text and calculations
- [ ] Design for the input field with left aligned label (See screendump from "oppstartskalkulator") 

## New design
**This should include**
- Create a new information component for presenting prefill or other information that the user cannot edit.
- Set what kind of color the field should have

**How a summary row should look like, instead of a deactivated input:** 

<img width="678" alt="image" src="https://user-images.githubusercontent.com/1464915/152972183-254d5867-e4b6-4230-9e98-e9e68487c8df.png">

---

**How data that is not editable should look like, instead of a deactivated input::** 

<img width="609" alt="image" src="https://user-images.githubusercontent.com/1464915/152973692-4f76c69e-a880-4184-8fdf-32cf959e12a4.png">

- When you are confirming something: 
<img width="1193" alt="image" src="https://user-images.githubusercontent.com/1464915/152972741-f3df5fcf-6661-41df-b84e-a7b6b65bbfce.png">

**Comment by RonnyB71** [comment]
@Febakke call in to a separate refinement meeting to clarify implementation and splitting into smaller issues.

Should a readonly field be a field in the model or should it be a computed text element .

**Comment by Febakke** [comment]
This issue is outdated
We have fixed our read-only components. Why you should try to avoid using read-only is handled in #207
